demo/tree-view-demo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { TreeView } from '../src';
+import { TreeItemData } from '../src/components/tree-view/tree-view';
+
+interface TreeViewDemoState {
```

Comment:
`type` suits better for State than `interface`. `interface` should be implemented, but `type` just describing a structure (shape) of an object.